Design-Docs/glossary.md

```diff
@@ -0,0 +1,143 @@
+
+It includes the Controller Nodes (Galera Cluster, RabbitMQ) and the Manager Nodes, Maas,...
+
+## Definition of Control Node
+
+The Control Node runs the Identity Service, Image Service , management processes for compute nodes, management processes for networking, various networking agents, and the Dashboard. It also includes supporting services such as an SQL database, a message queue, and NTP.
+
+Optionally, the Controller Node runs parts of the Block Storage, Object Storage, Orchestration and Telemetry services.
+
+The Controller Node requires at least two network interfaces.
+
+## Definition of Compute Node
+
+A compute host runs the hypervisor part of compute that runs instances. By default, compute uses the KVM hypervisor. The compute host also runs a networking service agent that connects instances to virtual networks and provides firewall services to the instances through security groups.
+
+If you offer hyper-converged infrastructure, a compute host also serves the Ceph. This makes the storage dynamically scalable (horizontally and vertically). For the Ceph services, 1 CPU core and 4 GB of RAM are reserved per OSD to ensure appropriate performance. 
+
+## Definition of Manager Node
+
+From here, the OSISM Ansible playbooks are applied to the environment. Furthermore, the following services often run here non-redundantly: Prometheus server,....
+
+## Definition of provider network
+
+The provider network is the network that is "in front", i.e. at the output points of the openstack/SCS. This is usually a public network, but can also be a private network in individual cases. IPs from the provider network can be assigned to instances within the SCS. The same applies to load balancers, of course. 
+
+## Definition of API
+
+The Rest API provides the core of openstack/SCS  and can be addressed for a whole region. It accepts and responds to end-user API calls.  The service supports the OpenStack Compute API, the Amazon EC2 API, and a special Admin API for privileged users to perform administrative actions. Policies are enforced and most orchestration actions can be started, such as launching an instance.
+
+## Horizon
+
+Horizon is openstack's preferred GUI for the end user, but also for the administrator for a quick overview. It runs on the controller node. 
+
+## Message Queue
+
+Most OpenStack services communicate with each other through the message queue. For example, Compute communicates with Block Storage services and Network services via the message queue. RabbitMQ, Qpid, and Zeromq are popular choices for a message queue service. When the message queue fails or becomes inaccessible, the cluster generally comes to a halt and ends up in a read-only state where the information is stuck at the point where the last message was sent. Therefore, this is clustered.  RabbitMQ has shown itself to be the most widespread and best supported variant in the OpenStack context, Qpid occurs occasionally, ZeroMQ lacks HA functionality to date
+
```

Comment:
Is 0MQ still supported? Can't remember if that removed from oslo_messaging.

Reply:
I think OMQ uses amqp, which is supported?
What is misssing here is kafka, which is pretty popular option for messaging at least outside of the openstack.